pkg/shell/superuser.jsx

```diff
@@ -229,7 +229,7 @@ const UnlockDialog = ({ proxy, host }) => {
         <Modal isOpen
                position="top"
                variant="medium"
-               onClose={D.close}
+               onClose={methods ? (busy ? cancel : D.close) : cancel}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19318-20230912-120833-e7583574-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)